packages/wrangler/package.json

```diff
@@ -116,5 +120,18 @@
     "setupFilesAfterEnv": [
       "<rootDir>/src/__tests__/jest.setup.ts"
     ]
+  },
```

Comment:
this doesn't seem right, that's why we have esbuild-jest, to strip the types.

Reply:
Because of this: https://github.com/aelbore/esbuild-jest/issues/57